data/set/finite.lean

```diff
@@ -287,4 +289,10 @@ assume hf,
 have injective (f ∘ (coe : ℕ → ℤ)), from injective_comp hf $ assume i j, int.of_nat_inj,
 not_injective_nat_fintype this
 
+instance set.fintype (α : Type u) [fintype α] : fintype (set α) :=
```

Comment:
Is this a double-take namespace: `set.set.fintype`?

Reply:
Fixed.